operations/enterprises.md

```diff
@@ -430,13 +440,38 @@ An object where keys are the [Language](configuration.md#language) codes and val
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `Id` | string | required | Unique identifier of the feature. |
+| `IsActive` | bool | required | Whether the space feature is still active. |
+| `Classification` | [Space feature classification](enterprises.md#space-feature-classification) | required | Classification of the feature. |
 | `Name` | string | required | Name of the feature. |
 | `Names` | [Localized text](enterprises.md#localized-text) | required | All translations of the name. |
 | `ShortName` | string | optional | Short name \(e.g. code\) of the feature. |
 | `ShortNames` | [Localized text](enterprises.md#localized-text) | required | All translations of the short name. |
 | `Description` | string | optional | Description of the feature. |
 | `Descriptions` | [Localized text](enterprises.md#localized-text) | required | All translations of the description. |
 
+#### Space feature classification
+
+* `AccessibleBathroom`
+* `AccessibleRoom`
+* `AirConditioning`
+* `Balcony`
+* `DoubleBed`
+* `ElevatorAccess`
+* `EnsuiteRoom`
+* `HighFloor`
+* `Kitchenette`
+* `LowerBed`
+* `OceanView`
+* `PrivateBathroom`
+* `PrivateJacuzzi`
+* `PrivateSauna`
+* `RiverView`
+* `RollawayBed`
+* `SharedBathroom`
+* `TwinBeds`
+* `UpperBed`
+* `SeaView`
```

Comment:
This is an incorrect definition of the space feature enum. SpaceFeature classification can be extended anytime. sou you need to put at the and
* ...
To show that it is not fixed enum.